.travis.yml

```diff
@@ -24,9 +24,13 @@ jobs:
         - mvn clean test
     - stage: deploy-scipooldev
       script:
+        - chmod +x ./pomSnapshotToGitHash.sh
+        - ./pomSnapshotToGitHash.sh
```

Comment:
should do `- ./pomSnapshotToGitHash.sh || travis_terminate 1` otherwise if the script fails the build will not.